llvm/test/MC/Xtensa/Options/code_density.s

```diff
@@ -0,0 +1,63 @@
+# RUN: llvm-mc %s -triple=xtensa -show-encoding --mattr=+density \
+# RUN:     | FileCheck -check-prefixes=CHECK,CHECK-INST %s
+
+.align	4
+LBL0:
+
+# Instruction format RRRN
+# CHECK-INST: add.n a2, a3, a4
+# CHECK: encoding: [0x4a,0x23]
+add.n a2, a3, a4
+
+# Instruction format RRRN
+# CHECK-INST: addi.n a2, a3, 3
+# CHECK: encoding: [0x3b,0x23]
+addi.n a2, a3, 3
```

Comment:
Would you please add a test case for imm=-1?

Reply:
I added test case.